docs/overview/architecture.md

```diff
@@ -1,10 +1,12 @@
-## LDK Architecture
-![LDK Architecture](../assets/ldk-architecture.svg)
+## Architecture
+![Architecture](../assets/ldk-architecture.svg)
```

Comment:
earlier this picture was .png in previous revision, its back to .svg i guess. 
svg are generally not considered standard and are security concern because they are executing code. 

"The danger of an SVG file comes from the fact that it’s an XML that can have embedded CSS and JavaScript. The web browser will automatically run any JavaScript embedded in an SVG file. Therefore, if the script contains malicious code, it will place the user’s computer at risk."
from: https://blog.online-convert.com/svg-file-and-its-danger/

Reply:
Although we created this image and its not a security concern, but would prefer if its just png or jpeg for simplicity.

Reply:
![LDK_architecture_revised_2022-08-17_opt](https://user-images.githubusercontent.com/2414177/185122575-85915dcb-f239-4a15-99b5-75094f87f6db.svg)

There is a slight issue with the dashed strokes in that image, so I fixed and optimized it. See above 73kb SVG.

> Although we created this image and its not a security concern, but would prefer if its just png or jpeg for simplicity.

While there is certainly the possibility that malicious code can be hidden inside an SVG file, the same can be said for any other pieces of code on the site.

Also, the existing architecture page contains 20 SVG tags in its current form. Including the current architecture image.

Therefore, I don't buy that argument at all @G8XSU.

Now having said that, if we want to have a discussion about switching to PNG or JPG for images like these, that conversation should be focused on usability and load time, IMO.

I ran this through a PNG optimizer and got it down to around 35kb. And we can get similarly good results with  a 2x export of the same image.

![LDK_architecture_revised_2022-08-17_opt](https://user-images.githubusercontent.com/2414177/185129407-41a816d7-499a-41c7-8eb1-72421da765aa.png)

Reply:
Mostly agree with @sbddesign here. Even if we want to have the performance discussion a 73kb optimized SVG is pretty good for this example and usage. I'll update to use the latest SVG provided for now and log an issue to discuss the overall strategy for image formats etc

Reply:
Well i was just pointing out that in previous rev it was png and now it changed to svg.
As long as it is intentional and not a miss, go ahead since we are uploading these to server :+1: